app/views/order_management/order_details/_print_details.html.haml

```diff
@@ -0,0 +1,38 @@
+.show-for-print
+  .banner-list
+    .row
+      %dl
+        %dt= @order_detail.class.human_attribute_name(:account)
+        %dd= @order_detail.account
+    - if @order_detail.project
+      .row
+        %dl
+          %dt= @order_detail.class.human_attribute_name(:project)
+          %dd= @order_detail.project.name
+    .row
+      = banner_label @order_detail, :order_status
+      = banner_date_label @order_detail, :reconciled_at if @order_detail.reconciled_at
+    .row
+      = banner_date_label @order_detail.reservation, :reserve_start_at, t("activerecord.attributes.reservation.reserve_start_date")
```

Comment:
Could we call `OrderDetail.human_attribute_name(:reserve_start_date)`? I feel like that might be a little cleaner.